src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -781,11 +787,37 @@ function loadOperation(
             IsContentType: isContentType,
             IsEndpoint: false,
             SkipUrlEncoding: false, //TODO: retrieve out value from extension
-            Explode: false,
-            Kind: kind
+            Explode:
+                (inputType as InputListType).ElementType &&
+                format &&
+                format === "multi"
+                    ? true
+                    : false,
+            Kind: kind,
+            ArraySerializationDelimiter: getArraySerializationDelimiter(format)
         } as InputParameter;
     }
 
+    function getArraySerializationDelimiter(
+        format: string | undefined
+    ): string | undefined {
+        if (!format) return undefined;
+        switch (format) {
```

Comment:
Can switch in js work on `undefined`? If so we do not need to check if format is undefined or not, it just fall into the case `default`

Reply:
updated